lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -2931,16 +2931,28 @@ fn test_inbound_reload_without_init_mon() {
 	do_test_inbound_reload_without_init_mon(false, false);
 }
 
-#[test]
-fn test_blocked_chan_preimage_release() {
+#[derive(PartialEq, Eq)]
+enum BlockedUpdateComplMode {
```

Comment:
The commit in `main` also has changes in `quiescence_tests.rs`, but I assume that file doesn't exist on 0.1. Could be mentioned in the commit message.